### ISSUE TITLE
Different ways of CSS generation between 2.X and 3.X

### ISSUE THREAD

**Issue by danielo515** [rekit/rekit#210]
Hello,
Thanks for this amazing feature.
I was in the middle of an app development when I realized there is a new version of rekit which has some important improvements, like using hooks and functional components.
However, after porting some components the CSS rules are being applied differently (but I'm using the savme MUI version...)

Here is the old version:
![image](https://user-images.githubusercontent.com/2270425/72221010-ded23880-3556-11ea-8c2b-5423e9671732.png)

This is using v3:

![image](https://user-images.githubusercontent.com/2270425/72221007-d4b03a00-3556-11ea-8e65-8820140d0896.png)

One key part seems to be the injection order of the CSS rules. On the new version they are injected first, while on the old version they are injected last.

Any reason for this change? What can I do to fix it?

**Comment by supnate** [comment]
Actually I was not aware of this change. Maybe it's from the change of create react app template.

**Comment by danielo515** [comment]
Yes, I thought that too. It can be the usage of webpack4 maybe... It is hard to determine, but is a real problem because you have to start using stuff like `!important` to beat the other rules.
